ThirdParty.json

```diff
@@ -0,0 +1,196 @@
+[
+  {
+    "name": "@zip.js/zip.js",
+    "license": [
```

Comment:
@ggetz Can we make these inline?

Reply:
We're using [`JSON.stringify`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify) to output formatted JSON. I'm not aware of an option that would keep arrays inline while still leaving the rest of the file formatted.

We could set up a plugin to format JSON like we do with JS files, but I think that's a bit out-of-scope for this PR.